src/sync.cpp

```diff
@@ -647,13 +648,12 @@ std::vector<SyncConfig> SyncConfigBag::all() const
 // new Syncs are automatically inserted into the session's syncs list
 // and a full read of the subtree is initiated
 Sync::Sync(UnifiedSync& us, const char* cdebris,
-           LocalPath* clocaldebris, Node* remotenode, bool cinshare, int ctag)
+           LocalPath* clocaldebris, Node* remotenode, bool cinshare, handle ctag)
```

Comment:
This last param is useless now. I'll remove it

Reply:
Fixed in https://github.com/meganz/sdk/pull/2462/commits/86b5baea24bdd550f6f4fcf2aeaf68f904f31cc5